src/client/app/components/about.ts

```diff
@@ -42,4 +69,30 @@ export class About {
     ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets)}
     ${storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)}
     `;
+
+    configs = [];
```

Comment:
I prefer variable initializations at the top of the file, above the constructor.  Otherwise it's hard to trace through where these things are declared.